lib/src/coq.rs

```diff
@@ -407,6 +408,36 @@ impl<'a> Expression<'a> {
                     ),
                 ]),
             ),
+            Self::MonadicApplication { func, args } => paren(
+                with_paren,
+                nest([
+                    func.to_doc(false),
+                    line(),
```

Comment:
```suggestion
                    text(" "),
```
as I feel it is better to keep the opening parenthesis on the same line.

Reply:
Ok